WordPress/Classes/ViewRelated/NUX/SplashPrologueView.swift

```diff
@@ -0,0 +1,43 @@
+import SwiftUI
+
+struct SplashPrologueView: View {
+
+    var body: some View {
+        ZStack {
+            Color(SplashPrologueStyleGuide.backgroundColor)
+            HStack {
+                Rectangle()
+                    .foregroundColor(.clear)
+                    .frame(minWidth: 0, maxWidth: .infinity)
+                Image("splashBrushStroke")
+                    .resizable()
+                    .aspectRatio(contentMode: .fill)
+                    .frame(minWidth: 0, maxWidth: .infinity)
+                    .padding(.trailing, 20)
+                    .foregroundColor(Color(SplashPrologueStyleGuide.BrushStroke.color))
+            }
+
+            VStack {
+                Image("splashLogo")
+                    .resizable()
+                    .frame(width: 50, height: 50)
+                    .padding(10)
+                Text(Self.caption)
+                    .multilineTextAlignment(.center)
+                    .font(SplashPrologueStyleGuide.Title.font)
+                    .foregroundColor(Color(SplashPrologueStyleGuide.Title.textColor))
+            }
+        }
+    }
+}
+
+private extension SplashPrologueView {
+    static let caption = NSLocalizedString(
```

Comment:
there's a bit of constants and magic numbers here and there, maybe we can group them into an enum or just in this extension?

Reply:
My thinking here was that I wanted to follow the pattern we used with `JetpackPrologueStyleGuide` and `JetpackPrologueViewController`. If we put everything into `SplashPrologueView ` we'd need to exclude the `primaryButtonStyle` and `secondaryButtonStyle` styles since it doesn't use them, so I don't think we can cleanly consolidate things.

Reply:
> One thing I noticed is that the `Jetpack` target does not build, likely there's just some target memberships to be fixed, but that should be it.

Thanks! At the last minute I thought I could slim things down by excluding some files from the Jetpack target, but didn't try to compile it afterwards. 😄 Fixed in 4b205f0e38822fe689dce61c59454750bfcebd2a.